Source/WebCore/platform/graphics/cocoa/AudioTrackPrivateWebM.cpp

```diff
@@ -106,6 +108,76 @@ std::optional<MediaTime> AudioTrackPrivateWebM::discardPadding() const
     return m_discardPadding;
 }
 
+String AudioTrackPrivateWebM::codec() const
+{
+    if (m_formatDescription) {
+        if (!m_formatDescription->codecString.isEmpty())
+            return m_formatDescription->codecString;
+        return String::fromLatin1(m_formatDescription->codecName.string().data());
```

Comment:
Is it worth setting   `m_formatDescription`?

Reply:
I'm not sure I understand the question?